libraries/AP_SurfaceDistance/AP_SurfaceDistance.cpp

```diff
@@ -0,0 +1,182 @@
+#include "AP_SurfaceDistance.h"
+
+#include <AP_RangeFinder/AP_RangeFinder.h>
+
+#ifndef RANGEFINDER_TIMEOUT_MS
+ # define RANGEFINDER_TIMEOUT_MS 1000        // rangefinder filter reset if no updates from sensor in 1 second
+#endif
+
+#if AP_RANGEFINDER_ENABLED
```

Comment:
Should be above `RANGEFINDER_TIMEOUT_MS`

Reply:
Need it outside for stale data function/check to still work.  Copter doesn't have a lot rangefinder defines in it and I am hesitant to go adding them in this pr.